test/webhook-proxy.test.ts

```diff
@@ -1,13 +1,19 @@
-const express = require('express')
+import express from 'express'
+import { Server } from 'http'
+import nock from 'nock'
+import { logger } from '../src/logger'
+import { createWebhookProxy } from '../src/webhook-proxy'
+
+// tslint:disable-next-line:no-var-requires
 const sse = require('connect-sse')()
-const nock = require('nock')
-const {createWebhookProxy} = require('../src/webhook-proxy')
-const {logger} = require('../src/logger')
 
 const targetPort = 999999
 
 describe('webhook-proxy', () => {
-  let app, server, proxy, url, emit
+  let app: express.Application
+  let server: Server
+  let proxy: any
```

Comment:
I don't believe `createWebhookProxy()` is typed, so I set this to `any` for now. Open to improvements in this PR if you have suggestions.